src/pager.h

```diff
@@ -186,6 +186,7 @@ class pager_t {
     bool is_navigating_contents() const;
 
     // Become fully disclosed.
+    bool get_fully_disclosed() const;
```

Comment:
this would be more consistent with others

```suggestion
    bool is_fully_disclosed() const;
```

also the comment is off now, can you move the new method in a separate paragraph like the other methods above?